data-structures-linked-lists/prepend.js

```diff
@@ -0,0 +1,16 @@
+/* global LinkedList */ // eslint-disable-line no-unused-vars
+/* exported prepend */
+
+function prepend(list, value) {
+  const newList = new LinkedList(value);
+  let current = list;
+  let currList = newList;
+
+  while (current) {
+    currList.next = new LinkedList(current.data);
+    current = current.next;
+    currList = currList.next;
+  }
+
+  return newList;
```

Comment:
This algorithm doesn't actually require a loop. Try to leave all of the existing nodes alone.